js/everything.js

```diff
@@ -433,9 +436,20 @@ function makeListenURL(videoID, videoPosition) {
  * @returns {string}
  */
 function makeSearchURL(searchQuery) {
-    var url = URI(window.location);
-    url.setSearch("q", searchQuery);
-    return url.toString().replace("#", "");
+    return cleanURL(window.location)
+      .setSearch("q", searchQuery)
+      .toString();
+}
+
+/**
+ * Remove any search params or fragments from a URL.
+ * @param url
+ * @returns {string} The stripped URL
+ */
+function cleanURL(url) {
```

Comment:
Maybe we just want to call `fragment("")`, do we need to clear the query args (assuming that's what `search("")`)?

Also, this should be moved above the first usage. Some browsers may error if it's used before it's declared. I know Firefox did this the last time I checked